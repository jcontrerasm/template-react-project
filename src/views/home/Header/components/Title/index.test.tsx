import * as React from 'react';
import { shallow } from 'enzyme';
import { Title } from './index';

describe('The Title component', () => {
  describe('Render snapshot', () => {
    it('Render correctly', () => {
      const wrapper = shallow(<Title text="hola"/>);
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
