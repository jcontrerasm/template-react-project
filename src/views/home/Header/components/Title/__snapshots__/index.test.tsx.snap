// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Title component Render snapshot Render correctly 1`] = `"<h1 class=\\"sc-bdVaJa bkvOLf\\">hola</h1>"`;
